foe-ios/Models/Sighting.swift

```diff
@@ -12,10 +12,11 @@ import GooglePlaces
 
 class Sighting {
   private var image: UIImage?
-  private var habitat: String?
+  private var habitat: String = ""
```

Comment:
is this initialization still necessary? looks like you handled the nil case on line 58 already!

Reply:
Right now the getter methods in `Sighting.swift` aren't consistent, so I think we need to standardize how we're initializing each of these member fields. It's either the members are:
- declared as optional using `?` and the `nil` case is handled in the get method or
- initialized with a default value and the get method simply returns the value

The first option is much more suitable in the case of checking the existence of each member:  checking if they're `nil` will be consistent across other files as opposed to checking if each member field for their particular empty value.